pkg/kube/inject/webhook_test.go

```diff
@@ -1045,11 +1045,12 @@ func BenchmarkInjectServe(b *testing.B) {
 	wh.Run(stop)
 
 	body := makeTestData(b, false, "v1beta1")
-	req := httptest.NewRequest("POST", "http://sidecar-injector/inject", bytes.NewReader(body))
-	req.Header.Add("Content-Type", "application/json")
 
 	b.ResetTimer()
 	for i := 0; i < b.N; i++ {
+		req := httptest.NewRequest("POST", "http://sidecar-injector/inject", bytes.NewReader(body))
```

Comment:
need to create a new request each time or we end up reading empty after the first time; reader is not re-usable